src/main/java/com/ge/predix/uaa/token/lib/AbstractZoneAwareTokenService.java

```diff
@@ -190,10 +193,22 @@ private void assertUserZoneAccess(final OAuth2Authentication authentication, fin
 
     protected FastTokenServices createFastTokenService(final List<String> trustedIssuers) {
         FastTokenServices tokenServices;
-        tokenServices = this.fastRemoteTokenServicesCreator.newInstance();
-        tokenServices.setStoreClaims(true);
-        tokenServices.setUseHttps(this.useHttps);
-        tokenServices.setTrustedIssuers(trustedIssuers);
+        try {
+            tokenServices = this.beanFactory.getBean(FastTokenServices.class);
```

Comment:
why get this bean from the context, if this class is going to override all properties anyway?  I suggest use new always.
Also, tokenService is private to this class,  and never expected to be configured externally.

Reply:
Got it. Since fastTokenservices is referenced/created in a local method scope in the ZacTokenService, would need to bring in powermock to write unit tests. I suspect the Creator might have been used to overcome testing challenges.